demos/system_reference-TI.rst

```diff
@@ -0,0 +1,54 @@
+.. _reference_board_TI:
+
+==================================
+Texas Instruments Reference Boards
+==================================
+
+A number of the `OpenAMP project examples <https://github.com/OpenAMP/openamp-system-reference/tree/main/examples>`_
+can be executed on TI Reference boards.
+
+The `Zephyr <https://www.zephyrproject.org/>`_ demo
+`OpenAMP using resource table <https://docs.zephyrproject.org/latest/samples/subsys/ipc/openamp_rsc_table/README.html>`_
+can be built for the following boards:
+
+.. csv-table::
+   :header: "Board", "Zephyr Identifier", "Deploy Firmware Name"
+   :widths: 50, 50, 50
+
+    `TI SK-AM64B <https://www.ti.com/tool/SK-AM64B>`_, ``sk_am64/am6442/m4``, ``am64-mcu-m4f0_0-fw``
```

Comment:
The second line in the table generated [here](https://openamp--54.org.readthedocs.build/en/54/demos/system_reference-TI.html#texas-instruments-reference-boards) has 3 lines instead of the expected 2.

![image](https://github.com/user-attachments/assets/e49127b2-a196-481f-b8e7-9712c239fb05)

Reply:
done